src/queries/rum-pageviews.sql

```diff
@@ -37,7 +37,7 @@ BEGIN
         WHEN 365 THEN TIMESTAMP_TRUNC(time, YEAR)
         ELSE TIMESTAMP_TRUNC(time, DAY)
       END AS date
-    FROM helix_rum.PAGEVIEWS_V3(
+    FROM helix_rum.PAGEVIEWS_V4(
```

Comment:
I did not create `PAGEVIEWS_V4` and I object strongly to using it in any of the default queries as it is hiding data.